backend/src/api/routes/organizationRoute.ts

```diff
@@ -0,0 +1,13 @@
+import { Request, Router } from 'express';
+import { run } from '../../common/helpers/routeHelper';
+import { createOrganization } from '../../services/organizationService';
+import { getUserOraganization } from '../../services/userOrganizationService';
+
+const router = Router();
+
+router
+  .get('/:orgId/user/:userId', run((req: Request) => (
```

Comment:
no need to set explicitly `req: Request` because the `req`'s type has been already set implicitly
```suggestion
  .get('/:orgId/user/:userId', run(req => (
```